spec/controllers/http_status_agent_spec.rb

```diff
@@ -36,6 +36,9 @@ def f.get url
 
                        def f.set url, response, time = nil
                          sleep(time/1000) if time
+                         def response.to_hash
```

Comment:
@cantino implemented `MockResponse`.

In principle I'm happy to try my hand at refactoring this spec, but I suspect I'm not experienced enough at Ruby to do a good job. So you're probably better off either giving me some pointers, or just doing it yourself :)

Either way is OK by me.